server/.gitignore

```diff
@@ -26,6 +26,7 @@ config/logging.json
 go.work
 go.work.sum
 
+.go
```

Comment:
What is this directory?

Reply:
We ou rmounting some local folders because the image we are using from chainguard to build mattermost has a readonly file system

Reply:
Specifically `cgr.dev/mattermost.com/go-msft-fips:1.22-dev`

Reply:
It's a temporary directory we mount within the build container